owncloudApp/src/main/java/com/owncloud/android/ui/activity/BiometricActivity.java

```diff
@@ -44,52 +50,113 @@
 import java.security.NoSuchProviderException;
 import java.security.UnrecoverableKeyException;
 import java.security.cert.CertificateException;
+import java.util.concurrent.Executor;
 
-public class FingerprintActivity extends AppCompatActivity {
-
-    private static final String TAG = FingerprintActivity.class.getSimpleName();
+public class BiometricActivity extends AppCompatActivity {
 
-    private static final String TAG_FINGERPRINT_FRAGMENT = "FINGERPRINT_LOCK";
+    private static final String TAG = BiometricActivity.class.getSimpleName();
 
     private static final String ANDROID_KEY_STORE = "AndroidKeyStore";
 
-    public final static String PREFERENCE_SET_FINGERPRINT = "set_fingerprint";
+    public final static String PREFERENCE_SET_BIOMETRIC = "set_fingerprint";
```

Comment:
You can modify set_fingerprint by the way.

`public final static String PREFERENCE_SET_BIOMETRIC = "set_biometric";`

Reply:
I didn't want to modify the sharedpref key so users who have the fingerprint lock enabled don't have to enable it again after updating the apk on PlayStore once this feature gets included. I haven't included it in the new commit, but please, let me know if I should change it.